src/modules/CMakeLists.txt

```diff
@@ -101,18 +100,15 @@ if( "${BACKEND}" STREQUAL "HIP")
     set(HIP_INC ${ROCM_PATH}/include/)
     list(APPEND HIP_INCLUDE_DIRS ${CMAKE_SOURCE_DIR}/src/include/hip/ ${CMAKE_SOURCE_DIR}/src/include/common/)
     set(INCLUDE_LIST ${HIP_INCLUDE_DIRS} ${HIP_INC} ${INCLUDE_LIST})
-    message("-- ${Blue}HIP include directories -- ${INCLUDE_LIST}${ColourReset}")
-
 elseif( "${BACKEND}" STREQUAL "OCL")
-
     # Add OpenCL kernels
     file(GLOB MOD_CL_CPP "cl/*.cpp" )
     list(APPEND Rpp_Source ${CPPFILES} ${MOD_CL_CPP})
     message("-- ${Green}OpenCL kernels added!${ColourReset}")
 
     # Set OpenCL compiler and flags
     set(CMAKE_CXX_COMPILER ${COMPILER_FOR_OPENCL}) # GCC and G++ donst work for creating .so file
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fopenmp -Ofast -msse4.2 -msse4.1 -mssse3 -mavx2 -mfma -g3 -std=c++14 -Wno-deprecated-declarations")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fopenmp -mavx2 -mfma -std=gnu++14")
```

Comment:
Would -mavx2 take care of all since rpp has both sse and avx code?

Reply:
@r-abishek yes, this would take care of it.